storybook/src/stories/Global/GlobalUI.stories.tsx

```diff
@@ -71,53 +156,99 @@ interface OwnProps {
-    introductionText= 'Call 00000000'
+    introductionText='Call 00000000'
   />
 )
 
-const Services = () => (
-  <div>
-  <PageHeader 
-    title='Services'
-    introductionText= 'We have plenty of services'
-  />
-    <ul>
-      <li><Link to='/services/custom'>Custom</Link></li>
-      <li><Link to='/services/special'>Special</Link></li>
-      <li><Link to='/services/extra-special'>Extra Special</Link></li>
-    </ul>
-  </div>
-
-);
-
 const ComponentLinks = () => (
-    <Switch>
-      <Route exact path="/welcome" component={Welcome} />
-      <Route exact path="/company" component={Company} />
-      <Route exact path="/company/about" component={About} />
-      <Route exact path="/company/team" component={Team} />
-      <Route exact path="/company/contact" component={Contact} />
-      <Route exact path="/support" component={Support} />
-      <Route exact path="/services" component={Services} />
-      <Route path="/services/:name" component={Service} />
-    </Switch>
+  <Switch>
+    <Route exact path="/welcome" component={Welcome} />
+    <Route exact path="/company" component={Company} />
+    <Route exact path="/company/about" component={About} />
+    <Route exact path="/company/team" component={Team} />
+    <Route exact path="/company/contact" component={Contact} />
+    <Route exact path="/support" component={Support} />
+    <Route exact path="/user/accounts" component={Accounts} />
+    <Route exact path="/user/billing" component={Billing} />
+    <Route exact path="/user/payments" component={Payments} />
+    <Route path="/services/:name" component={Service} />
+  </Switch>
 );
 
 
+const unreadAlerts: IAlertItem[] = [
+  {
+    imgUrl: '',
+    title: 'Event Type',
+    message: 'A short message limited to two lines. Extra text will just truncat...',
+    time: 'Just Now',
+  },
+  {
+    imgUrl: '',
+    title: 'Device is off',
+    message: 'The device has correctly turn off',
+    time: '1 min ago',
+  },
+  {
+    imgUrl: '',
+    title: 'Device is on',
+    message: 'The device has correctly turn on',
+    time: '6 mins ago',
+  },
+  {
+    imgUrl: '',
+    title: 'Connection was interrupted',
+    message: 'The connections is not working properly. Please verify your connection or contact your personal support.',
+    time: '1 hour ago',
+  },
+  {
+    imgUrl: '',
+    title: 'Device is off',
+    message: 'The device has correctly turn off',
+    time: '3 hour ago',
+  },
+]
+
+const readAlerts: IAlertItem[] = [
+  {
+    imgUrl: '',
+    title: 'Device is off',
+    message: 'The device has correctly turn off',
+    time: '3 days ago',
+  },
+  {
+    imgUrl: '',
+    title: 'Device is on',
+    message: 'The device has correctly turn on',
+    time: '3 days ago',
+  },
+  {
+    imgUrl: '',
+    title: 'A new device was added',
+    message: 'The device has bean correctly added',
+    time: '3 days ago',
+  }
+];
+
+const allAlerts: IAlertsHistory = {
+  unread: unreadAlerts,
+  read: readAlerts
+}
+
```

Comment:
Made read and unread alerts separately since I think this will be controlled and sorted by the Storybook User